fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -130,4 +160,17 @@ public String apply(final Integer type) {
 
     private static final Logger LOGGER = getLogger(DefaultMessageFactory.class);
 
+    private String getPropertiesAsString(final FixityEvent fixityEvent) {
+        final StringBuilder propsStrBuilder = new StringBuilder();
+        for (Map.Entry<String,String> entry : fixityEvent.getInfo().entrySet()) {
+            propsStrBuilder.append(entry.getKey()).append("->").append(entry.getValue()).append(",");
```

Comment:
@acoburn, I would argue that "Messaging" is a feature of F4. As such, probably a new child page of the following would be a good place to start documenting the details of F4 messages:
https://wiki.duraspace.org/display/FEDORA4x/Features